### PR TITLE
Use 攝 & 等 for deriving modern dialects

### LINKED ISSUE #73: 推導後世韻母改以「攝、等」爲條件
TshetUinh.js v0.15 更正了「攝」（元韻歸山攝）並且加入了「韻圖等」，以「攝」和 `音韻地位.韻圖等` 爲條件推導後世韻母不僅使書寫更簡明、更便於維護，而且在語義上更正確，因爲對後世音系來說，多數韻母之間的區別條件是韻圖音系的「攝、等」，而不是切韻音系的「韻、等」，逐個枚舉切韻韻目往往是多餘的

例如普通話推導方案：

https://github.com/nk2028/tshet-uinh-examples/blob/159d3ae830362628cde56addbc52b28b18c40cfa/putonghua.js#L106-L110

可以改爲：

``` javascript
  ['山攝', [
    ['一二等', [['合口', 'uan'], ['二等 牙喉音', 'ian'], ['', 'an']]],
    ['三四等', [['合口', 'üan'], ['', 'ian']]],
  ]],
```

原則上「等」也應該用 `音韻地位.韻圖等`，不過對於使用「切韻等」還是「韻圖等」在方案中不產生區別的情況，不妨直接從簡使用「切韻等」。上面普通話推導方案的片段就是例子：莊組切韻三等應是韻圖二等，無 i 介音，但是方案最終會將 zh 組後的 i 介音刪去，所以這裏用「切韻等」還是「韻圖等」最終不產生區別

對於方言中超韻圖音系的部分，即區分「重韻」，才以「韻」來判斷。例如廣州話部分區分蟹攝一等重韻：

https://github.com/nk2028/tshet-uinh-examples/blob/159d3ae830362628cde56addbc52b28b18c40cfa/gwongzau.js#L82-L86

---

目前需要修改的方案：

- 推導普通話 `putonghua.js`
- 推導廣州話 `gwongzau.js` 的部分攝
- 綾香思考音系 `ayaka_v8.js`
- 合入後的 #60 

應保持修改前後結果一致（除非改前的存在錯誤）

### LINKED ISSUE #75: 綾香思考音系「你」在「現代日語」模式下未轉化為「じ」
目前推導為「ぢ」

### PR DESCRIPTION
**Pull request by syimyuzya**
Resolves #73 & fixes #75

Other improvements on `putonghua` (35 音韻地位s affected):

- 雄 is now _xióng_ (previously _yóng_)
- 茝 is now _chǎi_ (previously _chǐ_)
- 銳 is now _ruì_ (previously same as 衛)
- 佳韻開口牙喉音 is _ie_ by default
  - （Temporary solution) _ai_ for 影溪母, _ia_ for 疑母 by default
- (Temporary solution) 尤韻脣音 (excluding 明母) is _u_ by default
  - 明母尤韻 is derived as 侯韻 (result same as before)
- 幽韻明母 is _miu_ (previously _miao_)
- 梗攝莊組三等入聲 is derived as _-e_ (previously _-i_)

(There are several places left for further improvements (chiefly regarding some minor cases and the 白讀 system), annotated with TODO and FIXME)

Other improvements on `gwongzau` (1 音韻地位 affected):
- 雄 is now _hung4_ (previously _jung4_)

Other improvements on `ayaka_v8` (28 音韻地位s affected):
- 四つ仮名 is now handled correctly

### CODE REVIEW

**Comment by syimyuzya** [comment]
三個方案都更新好了，再檢查一遍就可以合併了w

File: ayaka_v8.js [hunk 1]
```diff
@@ -169,167 +165,166 @@ function 韻母規則() {
...
-    ['模韻', [
-      ['脣音', 'o'],
-      ['舌齒音', 'o'],
-      ['影母', 'wo'],
-      ['牙喉音', 'o'],
+    ['遇攝', [
+      ['魚韻', [
+        ['莊組', 'o'],
+        ['舌齒音', 'yo'],
+        ['牙喉音', 'yo'],
```
Comment (untunt) [inline ayaka_v8.js:172]:
```suggestion
        ['莊組', 'o'],
        ['', 'yo'],
```
類似這種是不是都應該改成這樣？將「不發生音變」的環境逐個寫出是多此一舉，而且從原理上來說它是不能枚舉的

File: ayaka_v8.js [hunk 2]
```diff
@@ -169,167 +165,166 @@ function 韻母規則() {
...
-      ['脣音', 'ei'],
-      ['開口', 'ei'],
-      ['合口', 'wai'],
+    ['蟹攝', [
+      ['三四等', [
+        ['廢韻 平上聲 章組', 'ai'],
+        ['廢韻 合口', 'wai'],
+        ['脣音', 'ei'],
+        ['開口', 'ei'],
+        ['合口', 'wei'],
```
Comment (untunt) [inline ayaka_v8.js:197]:
```suggestion
        ['合口', 'wei'],
        ['', 'ei'],
```
這種更典型

File: ayaka_v8.js (hunk 1 above)
Comment (syimyuzya) [reply ayaka_v8.js:172]: > 而且從原理上來說它是不能枚舉的

對欸w

想起來這其實是曾經這些個方案依靠的「表格整理」的遺留【而且最近提到的那個 extrapolation-tool 就是輔助做整理表用的😂

File: ayaka_v8.js (hunk 2 above)
Comment (syimyuzya) [reply ayaka_v8.js:197]: 我把這一類都再檢查一遍吧w

**Review by untunt — APPROVED** [review]

**Pull request merged**